com.unity.netcode.gameobjects/Runtime/Core/NetworkObject.cs

```diff
@@ -796,7 +796,14 @@ internal void InvokeBehaviourNetworkSpawn()
             for (int i = 0; i < ChildNetworkBehaviours.Count; i++)
             {
                 ChildNetworkBehaviours[i].InternalOnNetworkSpawn();
-                ChildNetworkBehaviours[i].OnNetworkSpawn();
+                try
+                {
+                    ChildNetworkBehaviours[i].OnNetworkSpawn();
```

Comment:
User code. And I would argue the fact that we're a Unity product makes that expectation implicit. The engine doesn't exit when an event throws an exception, it just logs it and keeps going. By association with the engine, I think it's entirely reasonable for users to expect the netcode package to follow the same pattern as the rest of the product.

Reply:
Per our chat, it appears switching to `Debug.LogException()` here gives us what we want - same user look & feel for the error, the stack trace will point to the user's code as it would, yet we don't trigger runaway issues in our runtime